content/backmatter.tex

```diff
@@ -444,60 +434,61 @@ \subsection{SHMEM\_CACHE}
 \begin{center}
 \begin{tabular}{ll}
     \CorCpp: & \Fortran: \\
-    shmem\_clear\_cache\_inv & SHMEM\_CLEAR\_CACHE\_INV \\
-    shmem\_set\_cache\_inv & SHMEM\_SET\_CACHE\_INV \\
-    shmem\_set\_cache\_line\_inv & SHMEM\_SET\_CACHE\_LINE\_INV \\
-    shmem\_udcflush & SHMEM\_UDCFLUSH \\
-    shmem\_udcflush\_line & SHMEM\_UDCFLUSH\_LINE \\
-    shmem\_clear\_cache\_line\_inv \\
+    \FUNC{shmem\_clear\_cache\_inv}     & \FUNC{SHMEM\_CLEAR\_CACHE\_INV} \\
+    \FUNC{shmem\_set\_cache\_inv}       & \FUNC{SHMEM\_SET\_CACHE\_INV} \\
+    \FUNC{shmem\_set\_cache\_line\_inv} & \FUNC{SHMEM\_SET\_CACHE\_LINE\_INV} \\
+    \FUNC{shmem\_udcflush}              & \FUNC{SHMEM\_UDCFLUSH} \\
+    \FUNC{shmem\_udcflush\_line}        & \FUNC{SHMEM\_UDCFLUSH\_LINE} \\
+    \FUNC{shmem\_clear\_cache\_line\_inv} \\
 \end{tabular}
 \end{center}
-was originally implemented for systems with cache management instructions.
+was originally implemented for systems with cache-management instructions.
 This API has largely gone unused on cache-coherent system architectures.
 \FUNC{SHMEM\_CACHE} has been deprecated.
 
-\subsection{\_SHMEM\_* Library Constants}
+\subsection{\CONST{\_SHMEM\_*} Library Constants}
 The library constants
 \begin{center}
 \begin{tabular}{ll}
-    \_SHMEM\_SYNC\_VALUE & \_SHMEM\_REDUCE\_MIN\_WRKDATA\_SIZE \\
-    \_SHMEM\_BARRIER\_SYNC\_SIZE & \_SHMEM\_MAJOR\_VERSION \\
-    \_SHMEM\_BCAST\_SYNC\_SIZE & \_SHMEM\_MINOR\_VERSION \\
-    \_SHMEM\_COLLECT\_SYNC\_SIZE & \_SHMEM\_MAX\_NAME\_LEN \\
-    \_SHMEM\_REDUCE\_SYNC\_SIZE & \_SHMEM\_VENDOR\_STRING \\
+    \CONST{\_SHMEM\_SYNC\_VALUE}         & \CONST{\_SHMEM\_REDUCE\_MIN\_WRKDATA\_SIZE} \\
+    \CONST{\_SHMEM\_BARRIER\_SYNC\_SIZE} & \CONST{\_SHMEM\_MAJOR\_VERSION} \\
+    \CONST{\_SHMEM\_BCAST\_SYNC\_SIZE}   & \CONST{\_SHMEM\_MINOR\_VERSION} \\
+    \CONST{\_SHMEM\_COLLECT\_SYNC\_SIZE} & \CONST{\_SHMEM\_MAX\_NAME\_LEN} \\
+    \CONST{\_SHMEM\_REDUCE\_SYNC\_SIZE}  & \CONST{\_SHMEM\_VENDOR\_STRING} \\
 \end{tabular}
 \end{center}
 do not adhere to the \Cstd standard's reserved identifiers and the \Cpp
 standard's reserved names.  These constants have been deprecated and replaced
 with corresponding constants of prefix \shmemprefix{} that adhere to \CorCpp{}
 and \Fortran naming conventions.
 
-\subsection{\FUNC{shmem\_fetch}, \FUNC{shmem\_set}, \FUNC{shmem\_cswap},
+\subsection{\VAR{SMA\_*} Environment Variables}
```

Comment:
[Missing label from front matter conflict.](https://github.com/openshmem-org/specification/pull/120/files/238bc815d72d7f5bcc5ca6dc1655b1fc0922c3bb#diff-3b37f8686e2f1bb2e5cb250aa60d8d16)

Reply:
Whoops, missed that in the merge. I pushed a fix to the backmatter branch, so this should be resolved. I'll let RC1 stand for now and will double check for the undefined reference in the next build.